backbone.nativeajax.js

```diff
@@ -108,7 +108,7 @@
       }
 
       xhr.onreadystatechange = end(xhr, options, promise, resolve, reject);
-      xhr.open(options.type, options.url, true);
+      xhr.open(options.type, options.url, typeof options.async === 'undefined' || options.async);
```

Comment:
I prefer to pass arguments directly to native method as is, but seems in this case it does not mater.